modoboa_installer/scripts/amavis.py

```diff
@@ -17,7 +17,7 @@ class Amavis(base.Installer):
     packages = {
         "deb": [
             "libdbi-perl", "amavisd-new", "arc", "arj", "cabextract",
-            "liblz4-tool", "lrzip", "lzop", "p7zip-full", "rpm2cpio",
+            "lz4", "lrzip", "lzop", "p7zip-full", "rpm2cpio",
```

Comment:
Are you sure changing this package name has no impact on the other distributions?

Reply:
As far as I understand all those are Debian derivatives and the naming should be the same, I will create a few VMs with Debian and Ubuntu and verify.